connectors/sources/box.py

```diff
@@ -455,32 +462,29 @@ async def _consumer(self):
                 yield item
 
     async def get_docs(self, filtering=None):
-        already_processed_ids = set()
-        root_folder = "0"
-
+        stored_id = set()
         if self.is_enterprise == BOX_ENTERPRISE:
-            async for user_id in self.get_user_ids():
-                logger.debug(
-                    f"Fetching content for user with id '{user_id}' starting from root folder"
-                )
+            self._logger.info("Fetching data from Box's Enterprise Account")
+            async for user_id in self.get_users_id():
+                # "0" refers to the root folder
                 await self.fetchers.put(
-                    partial(self._fetch, doc_id=root_folder, user_id=user_id)
+                    partial(self._fetch, doc_id="0", user_id=user_id)
                 )
                 self.tasks += 1
         else:
-            logger.debug("Fetching content starting from root folder")
-            await self.fetchers.put(partial(self._fetch, doc_id=root_folder))
+            self._logger.info("Fetching data from Box's Free Account")
+            await self.fetchers.put(partial(self._fetch, doc_id="0"))
```

Comment:
I think having `"0"` be defined with a variable name `root_folder` (basically as it was before) is better for clarity. Someone without understanding of box will not know what `0` is at first glance. Could you revert that please?